test/reference.js

```diff
@@ -0,0 +1,13 @@
+var expect = require('chai').expect
+var reference = require('../1.0.0/reference.json');
+
+describe('1.0.0', function () {
+    describe('Should not have unsupported properties', function () {
+        it('markers.file', function () {
```

Comment:
These tests are not really adding much value... There're probably a lot of unsupported properties which we're not testing here... Is it worth testing a .json file? 😛